assets/scss/common/_custom.scss

```diff
@@ -72,14 +75,26 @@ h4 code {
   padding: 0 0.5rem;
 }
 
-#note.note-deprecated {
+#note.deprecated {
   background: lightyellow;
 }
 
-#note.note-notice {
+#note.notice {
   background: lightblue;
 }
 
-#note.note-warning {
+#note.warning {
   background: lightcoral;
 }
+
+[data-dark-mode] #note.deprecated {
+  background: yellowgreen;
+}
+
+[data-dark-mode] #note.notice {
+  background: darkcyan;
+}
+
+[data-dark-mode] #note.warning {
+  background: darkred;
```

Comment:
Would you be against making it more yellow / orange in dark mode?

Reply:
Hah I'm probably least qualified to pick colours, so anything you suggest I'll go with!

Reply:
Those look good to me, thanks for updating!